subprojects/gradle/code-ownership/src/main/kotlin/com/avito/android/CodeOwnershipExtension.kt

```diff
@@ -24,11 +29,11 @@ public open class CodeOwnershipExtension(
     private fun throwInvalidOwnershipSettingsException(projectPath: String) {
         throw IllegalStateException(
             """
-                |Owners must be set for $projectPath
-                |Configure ownership settings in $projectPath `build.gradle` file. For example: 
+                |Owners must be set for the $projectPath project.
+                |Configure the ownership extension for $projectPath in the `build.gradle.kts` file. For example: 
```

Comment:
```suggestion
                |Configure the ownership extension for $projectPath in the `build.gradle[.kts]` file: 
```
OR
```suggestion
                |Configure the ownership extension for $projectPath in the buildscript: 
```
It's an assumption about the type of file. We don't know it here.